charts/kubernetes-external-secrets/templates/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "kubernetes-external-secrets.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "kubernetes-external-secrets.name" . }}
+    helm.sh/chart: {{ include "kubernetes-external-secrets.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  selector:
+    app.kubernetes.io/name: {{ include "kubernetes-external-secrets.name" . }}
+  ports:
+    - protocol: TCP
+      port: {{ .Values.env.METRICS_PORT }}
+      name: prometheus
+      targetPort: {{ .Values.env.METRICS_PORT }}
```

Comment:
I believe this should be `prometheus` ? or is there some requirement for this to refer by value instead of the name used in the deployment?

Reply:
No requirement to refer to the value, both will work. If you prefer the name of the port, I will change it.